src/common/components/grid/grid.tsx

```diff
@@ -3,6 +3,7 @@ import React, { CSSProperties, useEffect, useState } from 'react';
 import { Box } from '@map-colonies/react-components';
 import { useTheme } from '@map-colonies/react-core';
 import { AgGridReact } from 'ag-grid-react';
+// import { ChangeDetectionStrategyType } from 'ag-grid-react/lib/changeDetectionService';
```

Comment:
Didn't see in **best-discretes.tsx** otherwise add not working properly.
Should be transfered via our's grid options in

Reply:
Done